fix-ssh-on-pi.ini

```diff
@@ -0,0 +1,5 @@
+root_password_encrypted="\$6\$SztiKXi5iDHrUgl7\$VChBXsW64HY3pSN/T4xpavMDAMdBvRDZXYUhwsMU5OGzKgSkLF3vJbpgapGriAse8dk9tTXAE8ctdoVXNGfb60"
+pi_password_encrypted="\$6\$SztiKXi5iDHrUgl7\$VChBXsW64HY3pSN/T4xpavMDAMdBvRDZXYUhwsMU5OGzKgSkLF3vJbpgapGriAse8dk9tTXAE8ctdoVXNGfb60"
```

Comment:
reviewer note: this encrypted password is the raspberry OS's default pw: `raspberry`